assets/models/system/salesforce-blip2-opt-2.7b/description.md

```diff
@@ -0,0 +1,44 @@
+`BLIP-2` is a model consisting of three components: a CLIP-like image encoder, a Querying Transformer (Q-Former), and a large language model. The image encoder and language model are initialized from pre-trained checkpoints and kept frozen while training the Querying Transformer. The model's goal is to predict the next text token given query embeddings and previous text, making it useful for tasks such as image captioning, visual question answering, and chat-like conversations. However, the model inherits the same risks and limitations as the off-the-shelf OPT language model it uses, including bias, safety issues, generation diversity issues, and potential vulnerability to inappropriate content or inherent biases in the underlying data. Researchers should carefully assess the safety and fairness of the model before deploying it in any real-world applications.
+
+> The above summary was generated using ChatGPT. Review the <a href='https://huggingface.co/Salesforce/blip2-opt-2.7b' target='_blank'>original-model-card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
+
+### Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href='https://aka.ms/azureml-infer-sdk-blip2-image-to-text' target='_blank'>image-to-text-online-endpoint.ipynb</a>|<a href='https://aka.ms/azureml-infer-cli-blip2-image-to-text' target='_blank'>image-to-text-online-endpoint.sh</a>
+Batch |<a href='https://aka.ms/azureml-infer-batch-sdk-blip2-image-to-text' target='_blank'>image-to-text-batch-endpoint.ipynb</a>|<a href='https://aka.ms/azureml-infer-batch-cli-blip2-image-to-text' target='_blank'>image-to-text-batch-endpoint.sh</a>
+
+### Sample inputs and outputs (for real-time inference)
+
+#### Sample input
+
+```json
+{
+   "input_data":{
+      "columns":[
+         "image"
+      ],
+      "index":[0, 1],
+      "data":[
+         ["image1"],
+         ["image2"]
+      ]
+   }
+}
+```
+Note:
+- "image1" and "image2" should be publicly accessible urls or strings in `base64` format.
+
+#### Sample output
```

Comment:
can we add an image similar to the other models so we make the model card more appealing?

https://ml.azure.com/models/microsoft-swinv2-base-patch4-window12-192-22k/version/9/catalog/registry/azureml?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rbhimani-rg/providers/Microsoft.MachineLearningServices/workspaces/rbhimani-eastus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

Reply:
added